### PR TITLE
feat: notifications toggle

### PR DESCRIPTION
**Pull request by lauti7**

### CODE REVIEW

File: src/components/Notifications/Notifications.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        />
+      </div>
+    )
+  })
+  .add('NotificationItemImage', () => {
+    return (
+      <div>
+        <NotificationItemImage
+          rarity={Rarity.EPIC}
+          imageLink="https://peer.decentraland.org/lambdas/collections/contents/urn:decentraland:ethereum:collections-v1:binance_us_collection:binance_us_upper_body/thumbnail"
```
Comment (braianj) [inline src/components/Notifications/Notifications.stories.tsx:228]:
```suggestion
          url="https://peer.decentraland.org/lambdas/collections/contents/urn:decentraland:ethereum:collections-v1:binance_us_collection:binance_us_upper_body/thumbnail"
```

File: src/components/Notifications/Notifications.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import { ActiveTab, DCLNotification, NotificationLocale } from './types'
+
+import NotificationBell from '../Icons/Notifications/NotificationBell'
+import NotificationBellActive from '../Icons/Notifications/NotificationBellActive'
+
+import './Notifications.css'
+
+export interface NotificationsProps {
+  isOpen: boolean
+  userNotifications: DCLNotification[]
```
Comment (braianj) [inline src/components/Notifications/Notifications.tsx:13]: What about?
```suggestion
  messages: DCLNotification[]
```

File: src/components/Notifications/Notifications.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  isLoading: boolean
+  locale: NotificationLocale
+  isOnboarding: boolean
+  activeTab: ActiveTab
+  onClickToggle: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
```
Comment (braianj) [inline src/components/Notifications/Notifications.tsx:18]: Let's continue with the standard
```suggestion
  onClick: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
```

File: src/components/Notifications/Notifications.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  userNotifications,
+  isLoading,
+  locale,
+  isOnboarding,
+  activeTab,
+  onClickToggle,
+  onChangeTab,
+  onBegin
+}: NotificationsProps) {
+  const unseenNotifications = userNotifications.filter(
```
Comment (braianj) [inline src/components/Notifications/Notifications.tsx:37]: Maybe a less complicated name
```suggestion
  const newNotifications = userNotifications.filter(
```

File: src/components/Notifications/NotificationsFeed.tsx [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+import './NotificationsFeed.css'
+import ItemSoldNotification from './NotificationTypes/ItemSoldNotification'
+import EmptyInbox from '../Icons/Notifications/EmptyInbox'
+import { Tabs } from '../Tabs/Tabs'
+import { Button } from '../Button/Button'
+import Time from '../../lib/time'
+import RoyaltiesEarnedNotification from './NotificationTypes/RoyaltiesEarnedNotification'
+
+interface NotificationsFeedProps {
+  userNotifications: DCLNotification[]
```
Comment (braianj) [inline src/components/Notifications/NotificationsFeed.tsx:15]:
```suggestion
  messages: DCLNotification[]
```

File: src/components/Notifications/NotificationsFeed.tsx [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+        newest: '最新',
+        read: '阅读'
+      },
+      empty: {
+        title: '你们都赶上了！',
+        description: '如果有新的通知，我们会及时通知您'
+      }
+    }
+  }
+}
```
Comment (braianj) [inline src/components/Notifications/NotificationsFeed.tsx:86]: Maybe we can put this into a global translation file

```suggestion
const i18N = {
  NotificationsFeed: {
     en: {
      onboarding: {
        title: 'Introducing Decentraland Inbox System',
        description:
          'Never miss anything anymore! Now you will get notified every time something relevant happens to your account.',
        button: "Let's begin"
      },
      feed: {
        title: 'Notifications',
        tabs: {
          newest: 'Newest',
          read: 'Read'
        },
        empty: {
          title: "You're all caught up!",
          description:
            "We'll let you know if there are new notifications for you."
        }
      }
    },
    es: {
      onboarding: {
        title: 'Presentacion de Decentraland Inbox',
        description:
          '¡No te pierdas nada nunca más! Ahora recibirás una notificación cada vez que ocurra algo relevante en tu cuenta.',
        button: 'Continuar'
      },
      feed: {
        title: 'Notificaciones',
        tabs: {
          newest: 'Mas reciente',
          read: 'Leidas'
        },
        empty: {
          title: '¡Ya estas al día!',
          description: 'Te avisaremos si hay nuevas notificaciones para ti.'
        }
      }
    },
    zh: {
      onboarding: {
        title: '介绍 Decentraland 收件箱系统 ',
        description:
          '不再错过任何信息！现在，每当您的账户发生相关事件，您都会收到通知。',
        button: '讓我們開始'
      },
      feed: {
        title: '通知',
        tabs: {
          newest: '最新',
          read: '阅读'
        },
        empty: {
          title: '你们都赶上了！',
          description: '如果有新的通知，我们会及时通知您'
        }
      }
    }
  }
}
```

File: src/components/Notifications/NotificationsFeed.tsx [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+    [userNotifications]
+  )
+
+  if (isOnboarding) {
+    return (
+      <div className="dcl notifications-feed">
+        <div className="dcl notifications-feed__content">
+          <div className="dcl notifications-feed__onboarding">
+            <div className="dcl notifications-feed__onboarding-bell"></div>
+            <p className="dcl notifications-feed__emptyview__title">
```
Comment (braianj) [inline src/components/Notifications/NotificationsFeed.tsx:148]: BEM should be like 
```suggestion
            <p className="dcl notifications-feed__empty-view-title">
```

File: src/components/Notifications/NotificationsFeed.tsx [hunk 8]
```diff
@@ -0,0 +1,253 @@
...
+                        locale={locale}
+                      />
+                    ))}
+                  </>
+                )}
+              </div>
+            </div>
+          )}
+          {!userNotifications.length && (
+            <div className="dcl notifications-feed__emptyview">
```
Comment (braianj) [inline src/components/Notifications/NotificationsFeed.tsx:238]: if you use the before suggestion change from `emptyview` to `empty-view` then this should be
```suggestion
            <div className="dcl notifications-feed__empty-view">
```

File: src/components/Notifications/NotificationsFeed.tsx [hunk 9]
```diff
@@ -0,0 +1,253 @@
...
+              <EmptyInbox />
+              <p className="dcl notifications-feed__emptyview__title">
+                {i18N[locale].feed.empty.title}
+              </p>
+              <p className="dcl notifications-feed__emptyview__description">
```
Comment (braianj) [inline src/components/Notifications/NotificationsFeed.tsx:243]: BEM should be like
```suggestion
              <p className="dcl notifications-feed__empty-view-description">
```

File: src/components/Notifications/NotificationTypes/ItemSoldNotification.tsx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+interface ItemSoldNotificationProps {
+  notification: ItemSoldNotification
+  locale: NotificationLocale
+}
+
+const i18N = {
+  en: { description: 'You just sold ', title: 'Item Sold' },
+  es: { description: 'Vendiste ', title: 'Item vendido' },
+  zh: { description: '你剛剛賣了 ', title: '已售商品' }
+}
```
[the remark this answers is not in the document]
Comment (lauti7) [reply src/components/Notifications/NotificationTypes/ItemSoldNotification.tsx:17]: For this particular case, I prefer to keep each i18N in each component file. I find it more comfortable and cleaner

File: src/components/Notifications/NotificationsFeed.tsx (hunk 8 above)
Comment (lauti7) [reply src/components/Notifications/NotificationsFeed.tsx]: keep it all as `notifications-feed__emptyview-*`

**Review by braianj — APPROVED** [review]
👍🏽

**Pull request merged**